roles/common/tasks/main.yml

```diff
@@ -25,6 +25,9 @@
       loop: "{{ common_centos8_repos }}"
```

Comment:
and these ones do not need removal? Just asking

Reply:
Those are the system repos, no need to remove them. We just want to remove the ones we download

Reply:
alright! makes sense